content/blog/2025/07/kubewarden-1.27-release.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Kubewarden 1.27 Release"
+authors:
+  - José Guilherme Vanz
+date: 2025-07-23
+---
+
+Here's a look at the key updates and improvements in the latest release.
+
+## New High-Risk Service Account Policy
+
+In this release, we've introduced a new policy to improve cluster security. The
+**High-Risk Service Account Blocker** policy, as its name suggests, blocks
+workloads that attempt to run with a service account that has excessive
+permissions.
+
+This policy allows cluster operators to define a list of forbidden permissions.
```

Comment:
I would like to have a small ClusterAdmissionPolicy example of the new policy to showcase it better.